service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ValidateOrderController.java

```diff
@@ -33,19 +32,6 @@
     private final ObjectMapper mapper;
     private final ValidateGroupService validateGroupService;
 
-    @PostMapping("/date-of-issue/mid-event")
-    public AboutToStartOrSubmitCallbackResponse handleMidEventValidateDateOfIssue(
```

Comment:
Do we still not use this for other orders, such as the orders in the create an order event?

Reply:
you're right we do, this could be a real messy problem. The mid event at the beginning of the event bulldozes anything you set in the about to start